core/app/models/spree/address.rb

```diff
@@ -9,7 +9,7 @@ class Address < Spree::Base
     has_many :cartons, inverse_of: :address
     has_many :credit_cards, inverse_of: :address
 
-    validates :firstname, :lastname, :address1, :city, :country, presence: true
+    validates :firstname, :lastname, :address1, :city, :country_id, presence: true
```

Comment:
Does this mean `validates :country_id` is the common ground for these concerns? It ensures a country was supplied, but without validating the contents of the Country instance.